backend/api/views/main.py

```diff
@@ -97,17 +97,28 @@ def populate_db():
     dates = list(files.values())
     dates.sort()
     date_initial = dates[0]
-    for file, date in files.items():
-        responses = parse_census_data(file, date, date_initial, parse2000)
+    objs = {}
```

Comment:
what's objects contain, can the dict have a more descriptive name or can u comment